scripts/Activity.vue

```diff
@@ -57,6 +58,12 @@ const txMap = computed(() => {
 async function update(fNewOnly = false, sync = true) {
     const cNet = getNetwork();
 
+    // If mempool is not loaded yet do not update the activity GUI
+    // or we might end up in a state where many of our addresses are considered invalid
```

Comment:
(In a future PR we could have a Wallet::isLoaded function that checks if all its components are loaded, instead of checking each thing individually like we're doing right now)